### PR TITLE
Document logging into an OpenShift cluster and location of kubectl config

### PR DESCRIPTION
**Pull request by rthallisey**
Fixes: #101 
Fixes: #100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,9 @@ where
 
 ## Install KubeVirt on an existing cluster
 
+Make sure you have a kubeconfig file that's available for ```kubectl``` to use.
```
Comment (lukas-bednar) [inline README.md]: Issue 100 was about missing kubectl command, I would mention in addition that `kubectl` has to be ensured.

**Comment by rthallisey** [comment]
@alexxa any more comments here?

**Comment by alexxa** [comment]
@rthallisey Yes, I mentioned a comment of @gbenhaim before: "We can add a test the check if kubectl exists, if not fail the playbook with an informative message." I think it can be a nice enhancement in addition to a doc note. Will you agree?

**Comment by rthallisey** [comment]
@alexxa I think a doc should be enough because the error message from #100  ```/bin/sh: kubectl: command not found``` is understandable that you don't have kubectl installed.

**Comment by alexxa** [comment]
@rthallisey Please update this PR, and move all the changes to playbooks/README.md

File: playbooks/README.md [hunk 2]
```diff
@@ -97,6 +96,10 @@ Currently we don't have a playbook which installs KubeVirt on a Kubernetes clust
 
 ### OpenShift cluster
 
+The playbook expects you have the client installed and a config file in
```
Comment (alexxa) [inline playbooks/README.md]: @rthallisey what client? and a link on installation step docs will be appreciated.

**Comment by alexxa** [comment]
Thank you! LGTM. Merging.

**Pull request merged**